### DISCUSSION TITLE
Can the summary be translated as well?

### DISCUSSION THREAD

**Discussion by eltabry** [bookfere/Ebook-Translator-Calibre-Plugin#239] [Ideas]
my dear
Thank you for translating metadata. Can the summary be translated as well?
It also remains to adjust the direction of the translated text to be from right to left, as it is in Google Translate

**Comment by bookfere** [comment]
Could you provide which element contains the “summary” you mentioned?

The direction issue is currently being addressed. You will be notified when the related issue is closed.

**Reply by eltabry** [reply]
...

**Reply by eltabry** [reply]
I noticed that when selecting metadata translation, it works well, and the selection continues to work even if it is disabled

**Reply by eltabry** [reply]
Could you provide which element contains the “summary” you mentioned?

i itried it i found it fixed? ..in newst Ver.
